examples/recording_service/pluggable_storage/c/HelloMsg_subscriber.c

```diff
@@ -0,0 +1,305 @@
+
```

Comment:
This file is exactly as the generated one, isn;t it? if so we should remove it

Reply:
Hmmmm, I know the publisher varies, but it's probably true that this is just the same as the generated one. I'll remove it.

Reply:
@titosanxez , I realised that in both subscribers we do modify the topic name. So I believe we cannot generate over them and we cannot remove them from git. That should be fine, though.